include/crocoddyl_msgs/inertial_parameters_subscriber.h

```diff
@@ -0,0 +1,156 @@
+///////////////////////////////////////////////////////////////////////////////
+// BSD 3-Clause License
+//
+// Copyright (C) 2020-2023, Heriot-Watt University, University of Oxford
+// Copyright note valid unless otherwise stated in individual files.
+// All rights reserved.
+///////////////////////////////////////////////////////////////////////////////
+
+#ifndef CROCODDYL_MSG_MULTIBODY_INERTIAL_PARAMETERS_SUBSCRIBER_H_
+#define CROCODDYL_MSG_MULTIBODY_INERTIAL_PARAMETERS_SUBSCRIBER_H_
+
+#include "crocoddyl_msgs/MultibodyInertialParameters.h"
+
+#include <mutex>
+#ifdef ROS2
+#include <rclcpp/rclcpp.hpp>
+#else
+#include <ros/node_handle.h>
+#endif
+
+typedef crocoddyl_msgs::MultibodyInertialParameters MultibodyInertialParameters;
+typedef const crocoddyl_msgs::MultibodyInertialParameters::ConstPtr
+    &MultibodyInertialParametersSharedPtr;
+typedef Eigen::Matrix<double, 10, 1> Vector10d;
+namespace crocoddyl_msgs {
+
+
+class MultibodyInertialParametersRosSubscriber {
+public:
+  EIGEN_MAKE_ALIGNED_OPERATOR_NEW
+
+   /**
+   * @brief Initialize the multi-body inertial parameters subscriber.
+   *
+   * @param[in] topic     Topic name
```

Comment:
```suggestion
   * @param[in] topic  Topic name (default: "/crocoddyl/inertial_parameters")
```

Please include this detail in the documentation for the other inertial classes (if relevant).